Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/VaccinationCertificateCard.kt

```diff
@@ -37,7 +37,7 @@ class VaccinationCertificateCard(parent: ViewGroup) :
 
         certificateDate.text = context.getString(
             R.string.vaccination_certificate_vaccinated_on,
-            certificate.vaccinatedOn.toShortDayFormat()
+            certificate.vaccinatedOn?.toShortDayFormat()
```

Comment:
this will display `null` right ? is this what is required ?

Reply:
yes, it displays "null", required is to not let the app crash

Reply:
Certificates used to be rejected when the dates where malformed -> no scan possible

Reply:
Updated according to the spec: 
`If any error occurs during formatting, the raw value shall be displayed.`

Reply:
But you are not providing the raw value here 
should not it be ?
```kotlin
certificate.vaccinatedOn?.toShortDayFormat() ?: certificate.rawCertificate.vaccination.dt
```

Reply:
sorry I somehow missed this one, fixed